packages/frontend/i18n/src/resources/en.json

```diff
@@ -619,6 +619,7 @@
   "com.affine.collection.helpInfo": "HELP INFO",
   "com.affine.collection.menu.edit": "Edit Collection",
   "com.affine.collection.menu.rename": "Rename",
```

Comment:
remove?

Reply:
`com.affine.collection.menu.rename` is used elsewhere